app/models/intake.rb

```diff
@@ -568,4 +576,13 @@ def new_dependent_token
     verifier = ActiveSupport::MessageVerifier.new(Rails.application.secret_key_base)
     verifier.generate(SecureRandom.base36(24))
   end
+
+  def compute_canonical_email_address
+    if email_domain == 'gmail.com'
```

Comment:
In theory, gmail.com is definitely not the only domain name with this treatment. If we were to try to do an excellent job at this, we'd do a DNS lookup on the email domain and check that it's operated by Google. That approach would probably mean this should be in a background job, and would probably mean that we'd want it in another table called like `IntakeCanonicalEmail` which `belongs_to :intake`, so we could fill it asynchronously and separately.

Related, we may want to canonicalize the gmail-equivalent domains -- @googlemail.com is treated the same as @gmail.com. See e.g. https://www.gmass.co/blog/domains-gmail-com-googlemail-com-and-google-com/